sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/ExtractKeyPhraseAsyncClient.java

```diff
@@ -74,73 +74,73 @@
             Objects.requireNonNull(document, "'document' cannot be null.");
             final TextDocumentInput textDocumentInput = new TextDocumentInput("0", document);
             textDocumentInput.setLanguage(language);
-            return extractKeyPhrases(Collections.singletonList(textDocumentInput), null)
-                    .map(keyPhraseResult -> {
-                        if (keyPhraseResult.isError()) {
-                            throw logger.logExceptionAsError(toTextAnalyticsException(keyPhraseResult.getError()));
+            return extractKeyPhrasesWithResponse(Collections.singletonList(textDocumentInput), null)
+                    .map(keyPhraseResultCollection -> {
+                        for (ExtractKeyPhraseResult keyPhraseResult : keyPhraseResultCollection.getValue()) {
```

Comment:
Can `keyPhraseResultCollection.getValue()` be null or empty?

Reply:
the valuable keyPhraseResultCollection, should change to response, so it will be response.getValue()

As I know, response.getValue() should not be empty or null?